### DISCUSSION TITLE
Start Tasmota Firmware Update via Homeassistant

### DISCUSSION THREAD

**Discussion by FaserF** [arendst/Tasmota#15480] [Q&A] [answered]
Homeassistant has a cool new feature, where integrations can display if its device has a firmware update available.
https://www.home-assistant.io/integrations/update/

Would it be possible to implement this with MQTT/ Tasmota Homeassistant integration?
Would be quite nice, to have this update sensor and start firmware updates through Homeassistant.

**Comment by barbudor** [comment, marked as answer]
Hi
This is totally independent from Tasmota.
HA Tasmota Integration is part of HA and I would advise to discuss this with the HA Tasmota Integration owners at HA.

**Reply by FaserF** [reply]
Thanks, done: https://github.com/emontnemery/hatasmota/issues/151
